src/sql/parts/dashboard/common/dashboardTab.contribution.ts

```diff
@@ -41,6 +43,11 @@ const tabSchema: IJSONSchema = {
 			description: localize('sqlops.extension.contributes.tab.when', 'Condition which must be true to show this item'),
 			type: 'string'
 		},
+		provider: {
+			description: localize('sqlops.extension.contributes.tab.provider', 'Defines the types of connection can this tab work with. Defaults to "MSSQL" if not set'),
```

Comment:
A little "nonstandard" sentence structure...perhaps `Defines the connection types this tab is compatible with...`  or similar.

Reply:
That's better, appreciate the help clarifying this.